src/main/java/com/example/springboot/model/CartItem.java

```diff
@@ -0,0 +1,37 @@
+package com.example.springboot.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE cart_items SET is_deleted = true WHERE id =?")
+@Where(clause = "is_deleted = false")
+@Table(name = "cart_items")
+public class CartItem {
+    @Id
```

Comment:
it's better to ident annotated fields
```suggestion
    @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    @ManyToOne
    @JoinColumn(name = "shopping_cart_id", nullable = false)
    private ShoppingCart shoppingCart;
    
    @ManyToOne
    @JoinColumn(name = "book_id", nullable = false)
    private Book book;
    
    @Column(nullable = false)
    private Long quantity;
```